src/scripts.js

```diff
@@ -22,7 +22,8 @@ let allRecipes;
 
 window.addEventListener('load', function() {
   console.log('page loaded 🥺');
-  allRecipes = new RecipeRepository(recipeData);
+  const recipeInstances = recipeData.map(recipe => new Recipe(recipe));
+  allRecipes = new RecipeRepository(recipeInstances);
```

Comment:
Other new instances of RecipeRepository will be favorites and weekly menu (recipes to cook), right?